### ISSUE TITLE
Add footer with link how to give feedback to website comment

### ISSUE THREAD

**Issue by felixfaassen** [open-zaak/open-zaak-website#44]
Add footer to the website and some instruction where people can provide feedback for the website.

**Comment by Ainali** [comment]
Good idea! Could we copy the way we have on the Foundation for Public Code [About](https://about.publiccode.net/) site where we  have a direct link to the file on GitHub and a prefilled issue link?
